bin/strings.json

```diff
@@ -136,11 +136,10 @@
         "COMPILE_ARG_OUTPUT" : "Specify the output directory.",
         "COMPILE_ARG_GROUP_ANDROID" : "Android Options",
         "COMPILE_ARG_AP" : "Specify the Android platform used for building Android apk.",
-        "COMPILE_ARG_NDK_MODE" : "Set the compiling mode of ndk-build, should be debug|release|none, native code will not be compiled when the value is none. Default is same value with '-m'.",
+        "COMPILE_ARG_NDK_MODE" : "Set the compiling mode of ndk-build, should be ndk|none, default is ndk. Native code will not be compiled when the value is none.",
```

Comment:
for the changes on Gradle config, the meaning of ndk-mode should be changed
```
# android native code build mode
# none, native code will never be compiled.
# cmake, native code will be compiled by CMakeLists.txt
# ndk, native code will be compiled by Android.mk
PROP_NDK_MODE=ndk
```
1. didn't support set `cmake` to `ndk-mode` yet
2. `-m mode`, to select the debug or release mode for every platform include android